### DISCUSSION TITLE
Is it possible to use editor.js with tinicms?

### DISCUSSION THREAD

**Discussion by ghost** [tinacms/tinacms#3196] [Help] [answered]
Hi everyone. 

*Is it possible to use editor.js with tinicms?*

#### Concept
![image](https://user-images.githubusercontent.com/111701513/190391808-e98093fd-b026-4b1b-9c7c-40e66fea3ed1.png)

**Comment by jamespohalloran** [comment, marked as answer]
You could theoretically create a editor.js Tina field component: https://tina.io/docs/extending-tina/custom-field-components/
I'm not aware of anyone who's tried this integration yet, so you'd be trailblazing a bit on this.

**Reply by ghost** [reply]
Hi jamespohalloran. so thank you for feedback... but... so what do you think of this idea? is it a good or bad idea?

**Reply by jamespohalloran** [reply]
Mind if I ask your main motivation for wanting to use editor.js with Tina? (Is it the UI? or the output format of the blocks?) I haven't used editor.js myself so it's difficult for me to weigh in too much. In terms of difficulty of the integration, this would likely be pretty difficult compared to a single field that does one function (e.g a "map picker" field.)

**Reply by ghost** [reply]
Hi jamespohalloran.
> Mind if I ask your main motivation for wanting to use editor.js with Tina? (Is it the UI? or the output format of the blocks?) 

Both - my goal would be UI and output format of the blocks.... but... why?

> I haven't used editor.js myself so it's difficult for me to weigh in too much. In terms of difficulty of the integration, this would likely be pretty difficult compared to a single field that does one function (e.g a "map picker" field.)

I agree with what you said here.
